languages/just/injections.scm

```diff
@@ -6,25 +6,26 @@
 
 ; ================ Always applicable ================
 
-((comment) @injection.content
-  (#set! injection.language "comment"))
+((comment) @content
```

Comment:
What is the point of setting the language of comments to "comment"? What sort of syntax highlighting would this provide? I know this existed before this PR but we might as well remove it if there isn't a reason to keep it.

Reply:
`comment` is not language supported by Zed, indeed.
I think the proper way would be to label this capture as `@comment`, a [default Zed capture](https://zed.dev/docs/extensions/languages#syntax-highlighting).

Reply:
Comments are already captured as you suggest in highlights.scm: https://github.com/jackTabsCode/zed-just/blob/bce8ce1c2212e4279315ca119153b38f20677c6c/languages/just/highlights.scm#L109. So I think we can just remove this comment language injection entirely yeah?

Reply:
I deleted it. Sorry for the delay.